src/core/OrbitControls.tsx

```diff
@@ -7,19 +7,20 @@ export type OrbitControlsProps = ReactThreeFiber.Overwrite<
   {
     target?: ReactThreeFiber.Vector3
     camera?: THREE.Camera
+    domElement?: HTMLElement
     regress?: boolean
     enableDamping?: boolean
   }
 >
 
 export const OrbitControls = React.forwardRef<OrbitControlsImpl, OrbitControlsProps>(
-  ({ camera, regress, enableDamping = true, ...restProps }, ref) => {
+  ({ camera, regress, domElement, enableDamping = true, ...restProps }, ref) => {
     const invalidate = useThree(({ invalidate }) => invalidate)
     const defaultCamera = useThree(({ camera }) => camera)
     const gl = useThree(({ gl }) => gl)
```

Comment:
couldn't we get the domElement from the gl?

Reply:
Yes, but passing it as a prop provides more flexibility... so you don't have to use the same domElement from the gl

Reply:
I think if no `domElement` is present it should use the canvas as default, or does it already do that in the implementation from `three-stdlib`?

Reply:
not 100% sure if it already does that but it should

Reply:
👍 Force pushed updated code

Reply:
🤔  Now that I think about it, I'm not sure that getting the `parentElement` is a smart approach. Would need to know whether `@react-three/fiber` was used via a <Canvas/> wrapper node or directly (without `react-dom` from v6).

Reply:
I think `gl.domElement` should probably be the default element @haywirez, this would resolve the requirement of knowing if someone used `<Canvas` or `render`

Reply:
OK, my latest change adds just that